kafka/integration_test.go

```diff
@@ -86,7 +87,7 @@ func runServer() (*testServer, error) {
 		"-p", "9092:9092",
 		"--env", "ADVERTISED_HOST=127.0.0.1",
 		"--env", "ADVERTISED_PORT=9092",
-		"spotify/kafka",
+		"uwdev/docker-kafka",
```

Comment:
Would it be worth considering something third party so we don't have to maintain it ourselves? For example, https://github.com/wurstmeister/kafka-docker.

Reply:
I think that's just a kafka image, so you need to run zookeeper in another container. The one we are using have both zookeeper and kafka in one image, so you only need to run one container.